lib/Driver/ToolChains/Gnu.cpp

```diff
@@ -2387,4 +2389,30 @@ void Generic_ELF::addClangTargetOptions(const ArgList &DriverArgs,
   if (DriverArgs.hasFlag(options::OPT_fuse_init_array,
                          options::OPT_fno_use_init_array, UseInitArrayDefault))
     CC1Args.push_back("-fuse-init-array");
+
+  if (getenv("HCC_BUILD")) {
+    CC1Args.push_back("-I" CMAKE_BUILD_INC_DIR);
+  }
+  else {
+    std::string path_hcc_include;
+
+    if (const char *p = getenv("HCC_HOME")) {
+      path_hcc_include = std::string(p) + std::string("/include");
+    }
+    else {
+      CC1Args.push_back("-I" CMAKE_ROCM_ROOT "/include");
+      path_hcc_include = getDriver().Dir + "/../../include";
+    }
+
+    HCCIncludePath = "-I" + path_hcc_include;
+    CC1Args.push_back(HCCIncludePath.c_str());
+  }
+
+  if (getenv("HCC_GTEST") && getenv("HCC_BUILD")) {
+    CC1Args.push_back("-I" CMAKE_GTEST_INC_DIR);
```

Comment:
Do we have to support HCC_GTEST in the compiler?

Reply:
I don't see anywhere we need this.

@whchung any idea why -gtest option was added to hcc-config?

Reply:
`gtest` was used in some ancient HCC unit tests IIRC. grep `gtest_amp` shows them.

Reply:
I don't see anywhere in test infrastructure that invokes `hcc-config --gtest` for example.  It looks like there's logic in tests/lit.cfg to add the required include path and libs already.